Makefile

```diff
@@ -6,3 +6,8 @@ lint:
 
 build:
 	go build -o dp3 ./client/dp3
+
+clean:
+	rm -rf data/*
+	rm -rf waldir/*
+	rm -rf wal.db*
```

Comment:
this db needs a new name now, probably dp3.db. It will be the rootmap + versionstore until we get those off sqlite.